scripts/generate_precompile.jl

```diff
@@ -34,6 +34,7 @@ toml = Pkg.TOML.parsefile(project_file)
 test_deps = get(toml, "extras", nothing)
 
 @info "Adding temporary dependencies ..."
+Pkg.activate(@__DIR__)
```

Comment:
I think this doesn't work, then we can't use dependencies of Atom.jl like `JSON` etc in `runtests.jl`.
This is why I choose to temporarily modify `Project.toml` of Atom.jl each time.